cli/ceph/osd/crush.py

```diff
@@ -0,0 +1,162 @@
+from cli import Cli
+
+
+class Crush(Cli):
+    """This module provides CLI interface to manage the MGR service."""
+
+    def __init__(self, nodes, base_cmd):
+        super(Crush, self).__init__(nodes)
+
+        self.base_cmd = f"{base_cmd} crush"
+
+    def create_rule(self, *args):
+        """
+        To create rules
+        Args:
```

Comment:
Change this to `Kargs`

Reply:
Here since the arguments needed for the create rule command doesn't need any key, passing them as key, value pair will not help us. And since we are not appending `--` with the args, we cannot use build_cmd_from_args method also. So I feel its better to use args itself.

Reply:
@pranavprakash20 yes, but need to correct description. Change `Args`  to `kwargs`  as they are dynamically received.

Reply:
Done